src/js/mock/S3Client.js

```diff
@@ -79,6 +90,27 @@ export const listBucketsResponse = (prefixWithSlash: string): ListObjectsRespons
     StartAfter: '',
 });
 
+export const userMetadata = {
+    'cache-control': '1',
```

Comment:
`cache-control` is one of the system metadata. Do you mind using `'color': 'green'` instead?

Reply:
done